### PR TITLE
Added One and Done to the QA options

### PR DESCRIPTION
**Pull request by rbillings**

### CODE REVIEW

File: locales/en.ini [hunk 1]
```diff
@@ -13,6 +13,7 @@ qa-intro = So you enjoy finding other programmers' mistakes?
 webqa-extra = find problems in Mozilla's websites
 ffqa-extra = help us find bugs in Firefox
 thunderbirdqa-extra = help us find bugs in Thunderbird
+oneanddone-extra = learn about Mozilla QA
```
Comment (jdm) [inline locales/en.ini]: This is pretty vague. Is there anything more specific we can put here that will help newcomers understand what it is they'll be linked to?

Comment (rbillings) [inline locales/en.ini]: agreed- I'll ponder on how to better describe a tool that covers all types of QA tasks

**Comment by rbillings** [comment]
r+?

**Comment by jdm** [comment]
Thanks!

**Pull request merged**